stellar-multitenant/pkg/tenant/manager.go

```diff
@@ -179,6 +179,11 @@ func (m *Manager) UpdateTenantConfig(ctx context.Context, tu *TenantUpdate) (*Te
 		args = append(args, *tu.Status)
 	}
 
+	if tu.DistributionAccount != nil {
+		fields = append(fields, "distribution_account = ?")
+		args = append(args, *tu.DistributionAccount)
+	}
+
```

Comment:
You should log a warning every time someone updates the distribution account public key. This is a pretty major change and we should highlight it.